api/lib/source/polkadotV0-source.js

```diff
@@ -1198,10 +1198,7 @@ class polkadotAPI {
       ),
       topVoters: await Promise.all(
         topVoters.map(async (topVoterAddress) => {
-          const accountInfo = await this.getNetworkAccountInfo(
-            topVoterAddress,
-            api
-          )
+          const accountInfo = await api.derive.accounts.info(topVoterAddress)
```

Comment:
why this change?

Reply:
because validators need to get more data from `identity` like web and so on, all fields that are not present in the `NetworkAccount` Object

Reply:
`topVoterReducer` doesn't use these infos. please double check

Reply:
ah true